opm/simulators/wells/BlackoilWellModelGasLift_impl.hpp

```diff
@@ -136,6 +136,10 @@ maybeDoGasLiftOptimize(const Simulator& simulator,
         num_wells_changed = glift_wells.size();
     }
     num_wells_changed = simulator.vanguard().gridView().comm().sum(num_wells_changed);
+
+    if (num_wells_changed > 0) {
+        updateWellPotentials(simulator, well_container, node_pressures, wellState, deferred_logger);
```

Comment:
This will only updates potentials for network nodes, right? Maybe add a comment about why potentials for other nodes are not updated? And why all network nodes need to be updated (and not only those that was changed)

Reply:
I have removed this and instead store the well potentials during gaslift to avoid re-computing the well potentials again.